src/libs/ReportUtils.js

```diff
@@ -895,6 +895,22 @@ function getPolicyExpenseChatName(report) {
     return reportOwnerDisplayName;
 }
 
+/**
+ * Get the title for a IOU or expense chat which will be showing the payer and the amount
+ *
+ * @param {Object} report
+ * @returns  {String}
+ */
+function getMoneyRequestReportName(report) {
+    const formattedAmount = NumberFormatUtils.format(preferredLocale, report.total / 100, {
+        style: 'currency',
+        currency: report.currency,
+    });
+
+    return Localize.translateLocal('iou.payerOwe',
+        {payer: isExpenseReport(report) ? getPolicyName(report) : getDisplayNameForParticipant(report.managerEmail), amount: formattedAmount});
```

Comment:
@mountiny @Julesssss Could you clarify if we are using the `managerEmail` or the `ownerEmail` for the display name?

Reply:
👋 from the doc @mollfpr, it's `manageEmail`. 

1. Create a new component AvatarWithDisplayName similar to HeaderView. However, it won’t have any of the pressable, navigation, error indicator and call button logic. 
   a. If isExpenseReport = true, the component will render SubscriptAvatar passing the user avatar as mainAvatar and the workspace avatar as secondaryAvatar.
   b. If isIOUReport = true, the component will render the avatar of the person who currently owes money, props.iouReport.managerEmail

Reply:
Agree with Tom. Thanks. 

The logic is a little tricky to understand at first glance, would you mind splitting it up into multiple lines?